plugins/outputs/cloudwatch/util_test.go

```diff
@@ -18,10 +18,16 @@ import (
 )
 
 func TestPublishJitter(t *testing.T) {
-	publishJitter := publishJitter(time.Minute)
-	log.Printf("Got publisherJitter %v", publishJitter)
-	assert.True(t, publishJitter >= 0)
-	assert.True(t, publishJitter < time.Minute)
+	// Loop an arbitrary number of times.
+	last := time.Duration(-1)
+	for i := 0; i < 100; i++ {
+		publishJitter := publishJitter(time.Minute)
+		log.Printf("Got publisherJitter %v", publishJitter)
+		assert.GreaterOrEqual(t, publishJitter, time.Duration(0))
+		assert.Less(t, publishJitter, time.Minute)
+		assert.NotEqual(t, publishJitter, last)
```

Comment:
There is effectively 0 change that 2 random numbers can be the same considering the range is 0 to 60,000,000,000.
If someone is opposed I can remove the `NotEqual` check.